### DISCUSSION TITLE
Feature request: HEIC support

### DISCUSSION THREAD

**Discussion by ericpardee** [DustinBrett/daedalOS#406] [Feature Requests]
<img width="573" alt="image" src="https://github.com/DustinBrett/daedalOS/assets/2423679/6ad74c67-8c7f-4047-9c35-18765e2970ae">

I think you can add HEIC image support using this library: https://github.com/catdad-experiments/heic-decode

**Comment by DustinBrett** [comment]
Thanks for the suggestion! I had https://github.com/ricokahler/heic.app bookmarked before but I think https://github.com/catdad-experiments/heic-decode also looks promising. I do indeed plan to add HEIC support. I actually noted this recently because Safari 17 added native support for it. https://caniuse.com/heif

I also now see https://github.com/catdad-experiments/libheif-js could be an even simpler dependency.

**Comment by DustinBrett** [comment]
Finally got around to adding this.

https://github.com/DustinBrett/daedalOS/commit/a65ebb0fc01a8f2089be98f7e4a31de73fb63e64

I used libheif-js in the end.

**Reply by DustinBrett** [reply]
I also added detection support to avoid loading this library for Safari which now supports HEIF natively.

https://github.com/DustinBrett/daedalOS/commit/b7b9538f143dd7991884861a1aceec0cb606e0ad
